x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/all_exception_items_table/index.tsx

```diff
@@ -0,0 +1,383 @@
+/*
```

Comment:
Tested locally and for all rules, the Rule Exceptions tab shows an infinite loading state:

<img width="1096" alt="Screenshot 2022-08-30 at 15 03 39" src="https://user-images.githubusercontent.com/1938181/187444081-3b79e0fd-3f6b-45e9-8b3f-6f02db590709.png">

And I don't see any outgoing requests in the Network tab. Is that part not implemented yet?

Reply:
There was a bug - it should be up and working now.